samples/cpp/benchmark_app/property_map.hpp

```diff
@@ -0,0 +1,172 @@
+    INSERT_PROPERTY_MAP(property_map, ov::intel_myriad::enable_receiving_tensor_time, PROPERTY_TYPE_BOOL, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_myriad::custom_layers, PROPERTY_TYPE_STRING, reversed);
+
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gna::firmware_model_image_path, PROPERTY_TYPE_STRING, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gna::pwl_max_error_percent, PROPERTY_TYPE_FLOAT, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gna::memory_reuse, PROPERTY_TYPE_BOOL, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gna::execution_mode, PROPERTY_TYPE_ENUM, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gna::execution_target, PROPERTY_TYPE_ENUM, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gna::compile_target, PROPERTY_TYPE_ENUM, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gna::pwl_design_algorithm, PROPERTY_TYPE_ENUM, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gna::scale_factors_per_input, PROPERTY_TYPE_STRING_FLOAT_MAP, reversed);
+
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gpu::context_type, PROPERTY_TYPE_ENUM, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gpu::ocl_context_device_id, PROPERTY_TYPE_INT, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gpu::tile_id, PROPERTY_TYPE_INT, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gpu::shared_mem_type, PROPERTY_TYPE_ENUM, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gpu::va_plane, PROPERTY_TYPE_UNSIGNED_INT, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gpu::enable_loop_unrolling, PROPERTY_TYPE_BOOL, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gpu::hint::queue_throttle, PROPERTY_TYPE_ENUM, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gpu::hint::queue_priority, PROPERTY_TYPE_ENUM, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gpu::hint::host_task_priority, PROPERTY_TYPE_ENUM, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_gpu::hint::available_device_mem, PROPERTY_TYPE_INT64, reversed);
+
+    INSERT_PROPERTY_MAP(property_map, ov::intel_myriad::hddl::graph_tag, PROPERTY_TYPE_STRING, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_myriad::hddl::set_stream_id, PROPERTY_TYPE_STRING, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_myriad::hddl::set_device_tag, PROPERTY_TYPE_STRING, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_myriad::hddl::bind_device, PROPERTY_TYPE_BOOL, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_myriad::hddl::runtime_priority, PROPERTY_TYPE_STRING, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_myriad::hddl::use_sgad, PROPERTY_TYPE_BOOL, reversed);
+    INSERT_PROPERTY_MAP(property_map, ov::intel_myriad::hddl::group_device, PROPERTY_TYPE_STRING, reversed);
+}
+
+void generate_enum_map(std::map<std::string, std::string>& enum_map) {
```

Comment:
Do we really want to use this non scalable way?
I think these tables will be out of date in the short time... I am not sure that developers will update these tables in case of new properties....